packages/e2e/.env.example

```diff
@@ -40,3 +40,6 @@ TX_SUBMIT_HTTP_URL=http://localhost:3456/tx-submit
 TRANSACTIONS_NUMBER=10
 START_LOCAL_HTTP_SERVER=true
 WORKER_PARALLEL_TRANSACTION=3
+
+# Required by test:providers, test:blockfrost
+STAKE_POOL_PROVIDER_URL="http://localhost:4000/stake-pool"
```

Comment:
Why can't it take the URL from the other variable though? Does the test need to use both variables at the same time?

Reply:
With last refactoring this is no longer used by blockfrost hybrid solution, this is used only by `yarn test:provider`.

`yarn test:provider` can be performed against a docker compose instance; in these case all the default values makes the test to work.

`yarn test:provider` can be performed against a deployed instance, in that case docker compose instance doesn't need to be started and `STAKE_POOL_PROVIDER_URL` needs to be set to point to the deployed instance API URL.

In the end `STAKE_POOL_PROVIDER_PARAMS` is used by `yarn test:wallet` and `STAKE_POOL_PROVIDER_URL` is used by `yarn test:provides`; they have two distinct meaning, the reason why they have the same value is to simplify running the tests against the docker compose instances (_local-network_, _preprod_ ect...)

To deduplicate this value we could do something similar to:

```
COMMON_STAKE_POOL_URL="http://localhost:4000/stake-pool"
STAKE_POOL_PROVIDER_PARAMS='{"baseUrl":"'$COMMON_STAKE_POOL_URL'"}'
STAKE_POOL_PROVIDER_URL=$COMMON_STAKE_POOL_URL
```

but in my opinion the lost readability doesn't worth the duplication removal.

Reply:
Sorry, but I still don't understand how they have a different meaning :sweat_smile: They both provide a stake pool provider URL :thinking: I understand that different test suites use different variables right now, but don't understand why can't the `yarn test:provider` just get `STAKE_POOL_PROVIDER_PARAMS` instead of `STAKE_POOL_PROVIDER_URL`